package.json

```diff
@@ -1,14 +1,25 @@
 {
-  "name": "chorus",
-  "private": true,
-  "type": "module",
-  "version": "1.22.2",
-  "scripts": {
-    "build": "rollup -c",
-    "watch": "rollup -c -w"
-  },
-  "devDependencies": {
-    "rollup": "3.28.1",
-    "rollup-plugin-chrome-extension": "3.6.12"
-  }
+    "name": "chorus",
+    "private": true,
+    "type": "module",
+    "version": "1.22.2",
+    "scripts": {
+        "build": "rollup -c",
+        "watch": "rollup -c -w",
+        "prepare": "husky"
+    },
+    "devDependencies": {
+        "husky": "^9.0.11",
+        "prettier": "^3.2.5",
```

Comment:
It would be great to use exact versions so every contributor is working with the exact same package version. Reference [save-prefix](https://pnpm.io/npmrc#save-prefix) to set the config key/value pair.

Reply:
@sstephanyy: Sorry, I might have not been clear. To ensure that packages use the exact same versions, we have to do the following:

1. Remove the prefix in front of the version. So from `"^9.0.11" to `"9.0.11"`. No `^` or `~` in front.
2. For future packages added, by default we also want pnpm to install & save the exact value. Therefore, we need to add an `.npmrc` file and set key-value of `save-prefix` to empty string. `save-prefix=''` or `save-prefix=""`. 


Thanks for making all these changes. This is the last tiny one. Otherwise, everything is looking good. 👍

Reply:
I’ve made the necessary changes, removing the version prefixes and configuring the .npmrc to ensure exact version installation of packages. I’ve learned a lot from this issue and am looking forward to working on another one (maybe the #207 issue?). Thank you for the positive feedback. It’s great to know everything is on track! 👍

Reply:
@sstephanyy : No problem. Glad to aid in your journey. Everything looks good and ready to merge. 
I'll assign #207 to you.